codecov.yml

```diff
@@ -0,0 +1,45 @@
+comment:
+  layout: newheader, reach, files, components, diff, flags # show component info in the PR comment
+  hide_comment_details: true  # hide the comment details (e.g. coverage targets) in the PR comment
+
+# Setting coverage targets
+coverage:
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 1%
+        removed_code_behavior: adjust_base
+    patch:
+      default:
+        target: 100%
```

Comment:
The thing that bothers me a bit with the current setup in `metadata-migration` is that the status is red when the patch coverage is not at 100%. But in practice it's super hard to get 100% coverage, and not necessarily essential for the project.

I don't know what is a good value here, or it can stay at 100%, just be aware of this.

Reply:
I am going to change it to 70% (5% below the current)
We plan to [increase the work](https://scality.atlassian.net/browse/COSI-62) coverage in the future